packages/sdk-ui-ts/src/utils/bridge.ts

```diff
@@ -263,6 +264,12 @@ export const tokenDenomsPerNetwork = [
     denoms: [],
     symbols: ['WMATIC'],
   },
+
+  {
+    network: BridgingNetwork.Noble,
+    denoms: ['uusdc'],
```

Comment:
put `ibc/2CBC2EA121AE42563B08028466F37B600F2D7D4282342DE938283CC3FB2BC00E` as the denom. `uusdc` is the cosmos baseDenom. full denom is `ibc/<hash>`